### PR TITLE
chore: extract button from core for error-boundary

### PR DESCRIPTION
**Pull request by gabriel-mirantis**

### CODE REVIEW

File: packages/core/src/extensions/renderer-api/components.ts [hunk 1]
```diff
@@ -36,7 +36,6 @@ export * from "../../renderer/components/layout/wizard-layout";
 export * from "../../renderer/components/layout/tab-layout";
 
 // form-controls
-export * from "../../renderer/components/button";
```
Comment (Nokel81) [inline packages/core/src/extensions/renderer-api/components.ts:39]: You need to replace this export with one from `@k8slens/button` to maintain backwards compatibility

File: packages/ui-components/tooltip/index.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
 export * from "./src/tooltip";
-export * from "./src/withTooltip";
\ No newline at end of file
+export * from "./src/withTooltip";
```
Comment (Nokel81) [inline packages/ui-components/tooltip/index.ts:2]: What is this change doing in this PR?

Comment (gabriel-mirantis) [reply packages/ui-components/tooltip/index.ts:2]: causing lint to fail less

File: packages/core/src/extensions/renderer-api/components.ts (hunk 1 above)
Comment (gabriel-mirantis) [reply packages/core/src/extensions/renderer-api/components.ts:39]: check

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**